client/coral-admin/src/routes/Moderation/components/Moderation.js

```diff
@@ -7,6 +7,7 @@ import ModerationMenu from './ModerationMenu';
 import ModerationHeader from './ModerationHeader';
 import ModerationKeysModal from '../../../components/ModerationKeysModal';
 import StorySearch from '../containers/StorySearch';
+import {isPremod, getModPath} from '../../../utils';
 
 export default class Moderation extends Component {
```

Comment:
The routing could should exist in the _container _. Here we only want to have _presentational_ related code.